From: =?utf-8?q?F=C3=A9lix_Sipma?= <felix+debian@gueux.org>
Date: Sun, 19 Apr 2020 16:43:08 +0200
Subject: Use local object inventory files for sphinx

Use the local object inventory files to prevent sphinx-build from
downloading them during build.
This patch makes the intersphinx mappings depend on the existence of
these files, thus passing Debian Policy 12.3 while still maintaining
a level of comfort for the reader of the documentation.
Based on Fladischer Michael <FladischerMichael@fladi.at> patch for
sphinx-issuetracker

Simplified to use plain paths to the packaged docs, not remote resources
by crusoe

Co-authored-by: Michael R. Crusoe <crusoe@debian.org>

Forwarded: not-needed
---
 docs/conf.py | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/docs/conf.py b/docs/conf.py
index 294c7dd..d315cac 100644
--- a/docs/conf.py
+++ b/docs/conf.py
@@ -47,9 +47,9 @@ autoapi_generate_api_docs = False
 # -- Options for intersphinx extension ---------------------------------------
 
 intersphinx_mapping = {
-    'jinja': ('https://jinja.palletsprojects.com/en/3.0.x/', None),
-    'sphinx': ('https://www.sphinx-doc.org/en/master/', None),
-    'python': ('https://docs.python.org/3/', None),
+    'jinja': ('/usr/share/doc/python-jinja2-doc/html', None),
+    'sphinx': ('/usr/share/doc/sphinx-doc/html/', None),
+    'python': ('/usr/share/doc/python3/html', None),
 }
 
 # -- Enable confval and event roles ------------------------------------------
